### PR TITLE
AND I MADE IT PERSONAL

### PR DESCRIPTION
**Pull request by b-maletckii**
LET ME IN,
LET ME IN

### CODE REVIEW

**Comment by claudiodsf** [comment]
I don't fully get what you did.

Could you please explain a little more??

File: README.md [hunk 1]
```diff
@@ -1,2 +1,43 @@
-# Billboard
-A collaborative editing exercise for the 2022 GIT class
+# MEH [BOI](https://youtu.be/KViOTZ62zBg) Production
```
Comment (claudiodsf) [inline README.md]: Really?? A YouTube link in the title?

**Comment by b-maletckii** [comment]
I MADE ONE OF THE MOST ICONIC README.md

TRUST ME

REAL MUSIC LOVERS WOULD APPRECIATE 

I  ADDED ALBUM REVIEW BUT MARKDOWN IS THE BEST THING I MADE IN 1.5-2 YEARS

**Comment by claudiodsf** [comment]
The content of README is nice but I would prefer it to be on a different markdown page

**Comment by b-maletckii** [comment]
> The content of README is nice but I would prefer it to be on a different markdown page

I hope it's better now

**Comment by b-maletckii** [comment]
@claudiodsf , what do you think now? Is it good enough to be considered to be pulled?

**Pull request merged**